bot.ml

```diff
@@ -786,15 +786,158 @@ let callback _conn req body =
                   ()
           else if
             string_match ~regexp:(f "@%s: [Mm]erge now" bot_name) body
-            && Option.is_some comment_info.pull_request
-          then
-            (* Should be restricted to coq *)
+            && comment_info.issue.pull_request
+            && String.equal comment_info.issue.issue.owner "coq"
+            && String.equal comment_info.issue.issue.repo "coq"
+          then (
+            (fun () ->
+              let pr = comment_info.issue in
+              GitHub_queries.get_pull_request_reviews_refs ~bot_info
+                ~owner:pr.issue.owner ~repo:pr.issue.repo
+                ~number:pr.issue.number
```

Comment:
We could apply all the tests that do not need this query first (labels, milestone, assignee, author) to spare a useless query in case they don't pass. We could also put the author test first, since it is not fixable. Regarding the tests that need this query, we could put the base branch test first, since it is the least fixable.